File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    let su = phone_number.split("");
    var answer = '';
    var max_num = su.length-4;
    let count = 0
    for(let i=0; i<su.length; i++){
        if(count>=max_num)
            answer+=su[i]
        else{
            count++;
            answer+="*"
        }
    }
    return answer;
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.py
def solution(phone_number):
    answer = ''
    max_num = len(phone_number)-4;
    count = 0
    for i in range(0,len(phone_number)) :
        if(count>=max_num) :
            answer+=str(phone_number[i])
        else : 
            count+=1
            answer+="*"
    return answer